### ISSUE TITLE
Rework CCM Setup Docs

### ISSUE THREAD

**Issue by ryanrichter94** [chocolatey/docs#329]
Wanna go through and rework the CCM setup docs. More clearly define the 6 options for install on all of the 3 CCM packages. Possibly provide recommended options to cut down on screen space/overload. Also cut down on the block quote usage as it makes the page very hard to follow.

**Comment by steviecoaster** [comment]
Good addition here is to ensure examples all use the same convention for package parameters (`:` as opposed to `=`)

**Comment by pauby** [comment]
The [Style Guide](https://docs.chocolatey.org/en-us/choco/commands/#scripting-integration-best-practices-style-guide) indicates we use `=`:

> When setting a value to an option, always put an equals (=) between the name and the setting, e.g. --source="'local'".

**Comment by pauby** [comment]
Actually, you said _package parameters_ which the Style Guide says:

> Now here is that example written with best practices (again line breaks added for readability - there are not line continuations for choco):
>
> `choco upgrade pkg1 -y --source="'https://community.chocolatey.org/api/v2'" --package-parameters="'/Option:Value /Option2:value with spaces'" --c4b-option="'Yaass'" --option-that-is-new="'dude upgrade'"`

So agreed 😄
